airbyte-cdk/python/airbyte_cdk/sources/declarative/models/declarative_component_schema.py

```diff
@@ -11,272 +11,272 @@
 
 
 class AuthFlowType(Enum):
-    oauth2_0 = 'oauth2.0'
-    oauth1_0 = 'oauth1.0'
+    oauth2_0 = "oauth2.0"
```

Comment:
This is interesting, I believe this file is generated, so we will probably want to tweak gen settings?